src/main/java/com/sopt/kwy/soptseminar/controller/user/dto/response/UserResDto.java

```diff
@@ -0,0 +1,16 @@
+package com.sopt.kwy.soptseminar.controller.user.dto.response;
+
+import com.sopt.kwy.soptseminar.domian.User;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
```

Comment:
```suggestion
@AllArgsConstructor(access = AccessLevel.PRIVATE)
```

public static 형태의 정적 팩토리 메서드만 사용해서 인스턴스를 생성할 경우에는
생성자를 private으로 접근하지 못하게 막아주는 것이 좋을 거 같네요!